dist/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Perfect Leaderboard</title>
+<script defer src="main.bundle.js"></script></head>
+<body>
+    <h1>Leaderboard!</h1>
+    <section class="hero">
+      <div class="container">
+        <div class="board">
+          <div class="head">
+            <h2>Recent Scores</h2>
+            <button class="refresh">Refresh</button>
+          </div>
+          <div class="scores"></div>
+        </div>
```

Comment:
- [ ] Great job on implementing the first step of your leaderboard project as specified in the requirements! 👏 However, I think it would be much better if you include a few hardcoded scores in your `HTML` as the project requirement ask that you write plain `HTML` to make the complete layout of the wireframe provided in the project requirement. Your scores should look something like below not completed empty.
<table>
<tr>
<th align=“middle”>Wireframe</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="219" alt="Screenshot 2023-07-25 at 22 45 03" src="https://github.com/mohamedgamaleldin999999/leaderboard-project/assets/107192076/cd3a9a5e-f820-4419-96af-b34045135f54">
</td>
<td>
<img width="317" alt="Screenshot 2023-07-25 at 22 45 24" src="https://github.com/mohamedgamaleldin999999/leaderboard-project/assets/107192076/ed8a0f4f-6373-4afd-a877-58789f9defcf">
</td>
</tr>
</table>